DSharpPlus.Test/Program.cs

```diff
@@ -13,19 +13,19 @@ public static void Main(string[] args)
 
         private async Task Run(string[] args)
         {
-            var cfg = TestBotConfig.Default;
+            var cfg = new TestBotConfig();
             var json = string.Empty;
             if (!File.Exists("config.json"))
             {
                 json = JsonConvert.SerializeObject(cfg);
-                File.WriteAllText("config.json", json, new UTF8Encoding(false));
+                File.WriteAllText(AppContext.BaseDirectory + Path.DirectorySeparatorChar + "config.json", json, new UTF8Encoding(false));
```

Comment:
Because `AppContext.BaseDirectory` does not necessarily point to the location of the executable. Similarly to `Environment.WorkingDirectory` in .NETFX.